aptos-move/aptos-vm/src/keyless_validation.rs

```diff
@@ -76,15 +81,19 @@ fn get_jwks_onchain(resolver: &impl AptosMoveResolver) -> anyhow::Result<Patched
 }
 
 pub(crate) fn get_groth16_vk_onchain(
+    features: &Features,
     resolver: &impl AptosMoveResolver,
+    module_storage: &impl AptosModuleStorage,
 ) -> anyhow::Result<Groth16VerificationKey, VMStatus> {
-    get_resource_on_chain::<Groth16VerificationKey>(resolver)
+    get_resource_on_chain::<Groth16VerificationKey>(features, resolver, module_storage)
 }
 
 fn get_configs_onchain(
+    features: &Features,
```

Comment:
Do we need to change this? IIRC the on chain config entry should not require resource group to store things together? This is probably not relevant to this PR per se but seems pretty complex to me.

Reply:
It does... Cryptography folks store they config as resource group...

Reply:
I also would love not to have resource groups as configs... that's why this code also not using `T::fetch_config(..)`